projects/ngrx.io/src/app/custom-elements/code/code.component.ts

```diff
@@ -17,7 +17,7 @@ import { tap } from 'rxjs/operators';
  * If linenums is not set, this is the default maximum number of lines that
  * an example can display without line numbers.
  */
-const DEFAULT_LINE_NUMS_COUNT = 10;
+const DEFAULT_LINE_NUMS_COUNT = 1_000;
```

Comment:
This caught my attention while review a PR.
I think the linenumbers take too much space and attention from the snippets, and I prefer to remove them because they don't add much value.
![image](https://github.com/ngrx/platform/assets/28659384/55352d6f-7165-4952-9d7f-6e3eb78fa531)